inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv4.cpp

```diff
@@ -0,0 +1,295 @@
+    jit.AddConstant(MakeJitConstant("BLOCK_SIZE", blockSize));
+    jit.AddConstant(MakeJitConstant("BLOCKS_COUNT", CeilDiv(params.output.X().v, blockSize)));
+
+    jit.Merge(MakeInputDeclsJitConstants(params, useVload8));
+    jit.Merge(MakeLoadJitConstants(params, useVload8));
+    jit.Merge(GetOperationsJitConstants(params, useVload8, vec_size));
+
+    std::string do_eltwise;
+    auto& operations = params.operations;
+    for (size_t op_num = 0; op_num < operations.size(); op_num++) {
+        for (size_t input_idx = 0; input_idx < params.inputs.size(); input_idx++) {
+            if (InputHasFeatureBroadcast(params, op_num, input_idx)) {
+                do_eltwise += "\\\n\tDO_FEATURE_BROADCAST" + std::to_string(op_num) + "_" + std::to_string(input_idx) + ";";
+            } else {
+                do_eltwise += "\\\n\tDO_VLOAD" + std::to_string(op_num) + "_" + std::to_string(input_idx) + ";";
+            }
+        }
+        do_eltwise += "\\\n\tOPERATION" + std::to_string(op_num) + ";";
+    }
+
+    do_eltwise += "\\\n\tres = tmp" + std::to_string(operations.size() - 1) + ";";
+
+    jit.AddConstant(MakeJitConstant("DO_ELTWISE", do_eltwise));
+
+    if (params.layoutBased || params.int8_quantization || params.broadcast) {
+        jit.Merge(GetTensorFriendlyWorkGroupsJit(params.output));
+    }
+
+    if (!params.stride.empty()) {
+        jit.AddConstant(MakeJitConstant("INPUT_STRIDED", 1));
+    }
+
+    jit.Merge(MakeActivationJitConstants(params.activations, params.output.GetDType(), "_TYPED"));
+
+    if (params.output.Feature().v % 4 != 0)
+        jit.AddConstant(MakeJitConstant("LEFTOVERS", params.output.Feature().v % 4));
+
+    if (!params.fused_ops.empty()) {
+        kernel_selector::Datatype input_dt = GetAccumulatorType(params);
+        std::vector<std::string> idx_order = {"b", "f_block*4", "y", "x"};
+        FusedOpsConfiguration conf = {"", idx_order, "res", input_dt, (size_t)vec_size};
+        conf.vec_axis = Tensor::DataChannelName::FEATURE;
+        jit.Merge(MakeFusedOpsJitConstants(params, {conf}));
+    }
+
+    jit.AddConstant(MakeJitConstant("ELTWISE_BROADCAST", params.broadcast));
+    jit.AddConstant(MakeJitConstant("QUANTIZATION_TERM", params.int8_quantization));
+    jit.AddConstant(MakeJitConstant("VEC_SIZE", vec_size));
+
+    return jit;
+}
+
+EltwiseKernelBase::DispatchData EltwiseKernel_b_fs_yx_fsv4::SetDefault(const eltwise_params& params) const {
+    DispatchData dispatchData;
+
+    dispatchData.gws[0] = params.output.X().v * params.output.Y().v;
+    dispatchData.gws[1] = CeilDiv(params.output.Feature().v, 4);
+    dispatchData.gws[2] = params.output.Batch().v;
+
+    dispatchData.lws= GetLocalWorkGroupSizes(dispatchData.gws, params.engineInfo);
+    dispatchData.lws[1] = 1;
+    dispatchData.lws[2] = 1;
+
+    return dispatchData;
+}
+
+// Local
+static std::vector<size_t> GetLocalWorkGroupSizes(std::vector<size_t> gws, const EngineInfo& info) {
```

Comment:
Is this same func in kernel_selector_utils.cpp (GetOptimalLocalWorkGroupSizes)

Reply:
Thank you. Removed.